src/utils/printing.jl

```diff
@@ -9,11 +9,12 @@ end
 function Base.show(io::IO, op_model::Simulation)
     println(io, "Simulation Model")
 end
-
- function Base.show(io::IO, res_model::OperationModelResults)
+ #=
```

Comment:
Please uncomment these lines, we need to develop proper show methods here but the code shouldn't be commented out when you make the PR.